README.md

```diff
@@ -76,6 +94,34 @@ settings.json:
 }
 ```
 
+## Npm Projects
+
+The extension builds the npm dependencies tree by running `npm list`. View licenses and top issue severities directly from package.json.
+To scan your npm dependencies, make sure to fulfill the following requirements:
+
+1. npm cli is installed in your system PATH.
+1. Your project is installed by running `npm install`.
```

Comment:
Important:
To have your project dependencies scanned by JFrog Xray, make sure the *npm CLI* is installed on your local machine and that it is in your system PATH.
In addition, the project dependencies must be installed using npm.